scripts/windows-download-cache-and-build-module-wheels.ps1

```diff
@@ -87,15 +99,14 @@ if (-not (Test-Path grep-win.zip)) {
 $env:Path += ";C:\P\grep"
 
 # Build ITK module dependencies, if any
-$build_command = "& `"C:\Python$pythonVersion-x$pythonArch\python.exe`" `"C:\P\IPP\scripts\windows_build_module_wheels.py`" --no-cleanup --py-envs `"3$($args[0])-x64`""
-foreach ($arg in $args[1..$args.length]) {
-  if ($arg.substring(0,2) -eq "--") {
-    $build_command = "$build_command $($arg)"
-  }
-  else {
-    $build_command = "$build_command `"$($arg)`""
-  }
+$build_command = "& `"C:\Python$pythonVersion-x$pythonArch\python.exe`" `"C:\P\IPP\scripts\windows_build_module_wheels.py`" --no-cleanup --py-envs `"3$python_version_minor-x64`""
+if ("$setup_options".length -gt 0) {
+  $build_command = "$build_command $setup_options"
+}
+if("$cmake_options".length -gt 0) {
+  $build_command = "$build_command -- ""$cmake_options"""
```

Comment:
Are extra quotes intended in this line? Probably the extra double quotes can be removed, or if they are required then they should be escaped with the ` character as done in line 102.

Reply:
Yes and they seem to be doing the same thing https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_quoting_rules?view=powershell-7.3. For consistency, I'll change them to backslash quotes now, thanks for noticing.

Reply:
You are right, thanks for the reference 👍